home.hbs

```diff
@@ -4,43 +4,82 @@
     <div class="gh-container">
         <div class="carousel slide" data-ride="carousel">
             <ol class="carousel-indicators">
-                <li data-target="#carouselIndicators" data-slide-to="0" class="active"></li>
+                <li
+                    data-target="#carouselIndicators"
+                    data-slide-to="0"
+                    class="active"
+                ></li>
```

Comment:
This style is pretty common in React/JSX, but seems pretty unnatural in "HTML-based" templates to me for some reason.

Reply:
I agree it's kinda unnatural but I think it makes sense for HTML because it tends to take up a lot of horizontal width due to indentations.